imports/features/surveys/services/emailService.ts

```diff
@@ -0,0 +1,103 @@
+import { Meteor } from 'meteor/meteor';
+import nodemailer from 'nodemailer';
+import { Email } from 'meteor/email';
+
+// Create a transporter with Mailtrap SMTP credentials
+export const createTransporter = () => {
+  // Check if we have settings, otherwise use defaults for development
+  const settings = Meteor.settings?.private?.mailtrap || {
+    host: 'sandbox.smtp.mailtrap.io',
+    port: 2525,
+    user: process.env.MAILTRAP_USER || 'default_user',
+    pass: process.env.MAILTRAP_PASS || 'default_pass'
+  };
```

Comment:
@amit7046  please can you make these values be able to be sent from the .env.

Basically, looking at:

SMTP_EMAIL_HOST
SMTP_EMAIL_PORT
SMTP_EMAIL_USER
SMTP_EMAIL_PASSWORD